# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/svgalib/ppcmemset-size_t_not_int-return_voidptr.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur svgalib-1.9.19-orig/src/ppcmemset.h svgalib-1.9.19/src/ppcmemset.h
--- svgalib-1.9.19-orig/src/ppcmemset.h	2005-01-06 16:32:09.938772576 +0100
+++ svgalib-1.9.19/src/ppcmemset.h	2005-01-06 16:41:14.163037912 +0100
@@ -1,7 +1,7 @@
 #ifdef __PPC
 
 #undef memset
-static inline void memset(void *a, int c, int s)
+static inline void *memset(void *a, int c, size_t s)
 {
    int i;
    for(i=0;i<s;i++) *((unsigned char *)a + i) = c;
